binzx/otomi

```diff
@@ -326,6 +325,7 @@ else
     "$otomi_tools_image" \
     bash -c "$cmd"
   status=$?
+  rm -rf $PWD/env &>/dev/null
```

Comment:
Why removing env dir ?

Reply:
it is the link that gets created either by symlinking or the docker runner that leaves it behind. This just removes it

Reply:
If so, then please use `unlink $PWD/env`

Reply:
Sometimes it is seen as a folder (docker remnants), sometimes as a link (when created as a link in dev mode).

Reply:
I still do not understand why we remove content of this directory. These could be a values mounted from users filesystem. Or I misundersand sth?